index.html

```diff
@@ -68,219 +63,246 @@ <h1>
     </section>
 
     <main class="grid">
+      <!-- First section starts -->
       <section class="work-one work">
         <img class="img-work" src="./templates/Snap1(4).svg" alt="Tonic" />
-        <h2>Multi-Post<br />Stories</h2>
-        <ul class="work-history">
-          <li class="cover">CANOPY</li>
-          <li>
-            <img
-              class="counter"
-              src="./templates/Counter(1).svg"
-              alt="counter"
-            />
-          </li>
-          <li class="career">back end dev</li>
-          <li>
-            <img
-              class="counter"
-              src="./templates/Counter(1).svg"
-              alt="counter"
-            />
-          </li>
-          <li class="career">2015</li>
-        </ul>
-        <p class="body-detail">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
-        </p>
-        <ul class="courses">
-          <li>html</li>
-          <li>css</li>
-          <li>javaScript</li>
-        </ul>
-        <button class="btn-body" type="submit" name="submit">
-          see project
-        </button>
+        <div class="body-content">
+          <h2>Facebook 360</h2>
+          <ul class="work-top">
+            <li class="canopy">Facebook</li>
+            <li>
+              <img
+                class="counter"
+                src="./templates/Counter(1).svg"
+                alt="counter"
+              />
+            </li>
+            <li class="dev">Back End Dev</li>
```

Comment:
- [ ] Let us align all these elements horizontally instead of vertically.
![Screenshot from 2021-08-28 20-11-28](https://user-images.githubusercontent.com/65501535/131225537-52c871b8-8dd8-4e95-8b7a-89b181ff5545.png)